nrich-spring-boot/src/main/java/net/croz/nrich/springboot/condition/OnPropertyNotEmptyCondition.java

```diff
@@ -21,7 +21,7 @@ public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata)
         Environment environment = context.getEnvironment();
         String propertyName = (String) Objects.requireNonNull(attributes).get(PROPERTY_NAME_ANNOTATION_VALUE);
 
-        String stringPropertyValue = environment.getProperty(propertyName);
+        String stringPropertyValue = bindWithExceptionIgnored(environment, propertyName, String.class);
```

Comment:
Not sure about the commit message for this change. Maybe something like `Resolve String property in a safe way` (`also` seems unnecessary to me) or `Resolve String property with exception ignored`.